bower.json

```diff
@@ -25,6 +25,10 @@
     "apis"
   ],
   "dependencies": {
-    "core-shared-lib": "Polymer/core-shared-lib#^0.5.5"
+    "polymer": "Polymer/polymer#0.8-preview",
+    "iron-jsonp-library": "PolymerElements/iron-jsonp-library#^0.8.0",
```

Comment:
It needs a 0.8.0 tag (https://github.com/PolymerElements/iron-jsonp-library/releases) for bower to be able to pull that version.